airbyte-webapp/src/packages/cloud/views/auth/components/PersonQuoteCover/PersonQuoteCover.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import { FormattedMessage } from "react-intl";
+
+import styles from "./PersonQuoteCover.module.scss";
+
+export interface PersonQuoteCoverProps {
+  overlayGradientStyle?: React.CSSProperties;
+  backgroundImageStyle?: React.CSSProperties;
+  quoteText?: string;
+  quoteTextStyle?: React.CSSProperties;
+  logoImageSrc?: string;
+  quoteAuthorFullNameStyle?: React.CSSProperties;
+  quoteAuthorJobTitleStyle?: React.CSSProperties;
```

Comment:
Most likely not needed with our discussion how to solve that, @edmundito. But would you be fine to just remove that in your PR again, or should we try to clean this PR up for things we'll need later on?

Reply:
I would prefer to remove them once a solution is in place in a separate PR.